### PR TITLE
new devscreen nav

### PR DESCRIPTION
**Pull request by GantMan**
This is using `react-navigation` and stays true to @justinhuskey and @cindy-nguyen 's design.

new_nav

### CODE REVIEW

File: packages/ignite-dev-screens/templates/ComponentExamplesScreen.js.ejs [hunk 1]
```diff
@@ -27,6 +27,14 @@ class ComponentExamplesScreen extends React.Component {
     return (
       <View style={[styles.container, styles.mainContainer]}>
         <Image source={Images.background} style={styles.backgroundImage} resizeMode='stretch' />
+        <TouchableOpacity onPress={()=> this.props.navigation.goBack()} style={{
+          position: 'absolute',
+          paddingTop: 30,
+          paddingHorizontal: 5,
+          zIndex: 10
```
Comment (skellock) [inline packages/ignite-dev-screens/templates/ComponentExamplesScreen.js.ejs]: I will fight you to the death!!!!

File: packages/ignite-dev-screens/templates/DeviceInfoScreen.js.ejs [hunk 2]
```diff
@@ -128,6 +128,14 @@ export default class DeviceInfoScreen extends React.Component {
     return (
       <View style={styles.mainContainer}>
         <Image source={Images.background} style={styles.backgroundImage} resizeMode='stretch' />
+        <TouchableOpacity onPress={()=> this.props.navigation.goBack()} style={{
+          position: 'absolute',
+          paddingTop: 30,
+          paddingHorizontal: 5,
+          zIndex: 10
```
Comment (skellock) [inline packages/ignite-dev-screens/templates/DeviceInfoScreen.js.ejs]: Again. We fight. To the death!!!

File: packages/ignite-dev-screens/templates/PluginExamplesScreen.js.ejs [hunk 3]
```diff
@@ -18,6 +18,14 @@ class PluginExamplesScreen extends React.Component {
     return (
       <View style={styles.mainContainer}>
         <Image source={Images.background} style={styles.backgroundImage} resizeMode='stretch' />
+        <TouchableOpacity onPress={()=> this.props.navigation.goBack()} style={{
+          position: 'absolute',
+          paddingTop: 30,
+          paddingHorizontal: 5,
+          zIndex: 10
```
Comment (skellock) [inline packages/ignite-dev-screens/templates/PluginExamplesScreen.js.ejs]: For the 3rd time.  It is death, to which we fight!

File: packages/ignite-dev-screens/templates/PresentationScreen.js.ejs [hunk 4]
```diff
@@ -1,39 +1,54 @@
...
 
   render () {
     return (
       <View style={styles.mainContainer}>
         <Image source={Images.background} style={styles.backgroundImage} resizeMode='stretch' />
+        <TouchableOpacity onPress={this.props.screenProps.toggle} style={{
+          position: 'absolute',
+          paddingTop: 30,
+          paddingHorizontal: 10,
+          zIndex: 10
```
Comment (skellock) [inline packages/ignite-dev-screens/templates/PresentationScreen.js.ejs]: To the death!!!

File: packages/ignite-dev-screens/templates/ThemeScreen.js.ejs [hunk 5]
```diff
@@ -57,6 +57,14 @@ export default class ThemeScreen extends React.Component {
     return (
       <View style={styles.mainContainer}>
         <Image source={Images.background} style={styles.backgroundImage} resizeMode='stretch' />
+        <TouchableOpacity onPress={()=> this.props.navigation.goBack()} style={{
+          position: 'absolute',
+          paddingTop: 30,
+          paddingHorizontal: 5,
+          zIndex: 10
```
Comment (skellock) [inline packages/ignite-dev-screens/templates/ThemeScreen.js.ejs]: WE FIGHT!  THIS TIME TO THE DEATH!

**Comment by GantMan** [comment]
@skellock  long story long.  I've absolutely positioned the navigation elements, because I can't make the navigation header transparent.

**Comment by skellock** [comment]
At a quick glance, I don't know how it'll be done any other way without opening up `react-navigation`.

**Pull request merged**